### PR TITLE
added description

### PR DESCRIPTION
**Pull request by yeongjukang**
Hi, @savsgio .
I added some descriptions for users to understand code clearly, with emojis taken out.
Updated codes cover 'jwt_auth' and 'static_files' this time.

### CODE REVIEW

File: static_files/main.go [hunk 1]
```diff
@@ -22,6 +40,12 @@ func main() {
 		After:  []atreugo.Middleware{afterView},
 	}
 
+	server.GET("/", func(ctx *atreugo.RequestCtx) error {
+		defer logger.Info("\t\tviewFn's defer")
```
Comment (savsgio) [inline static_files/main.go]: could you change `viewFn` by `view`? please

File: static_files/middlewares.go [hunk 2]
```diff
@@ -6,25 +6,25 @@ import (
...
 func beforeView(ctx *atreugo.RequestCtx) error {
-	logger.Info("Middleware executed BEFORE VIEW")
+	logger.Info("\tMiddleware executed BEFORE SPECIFIC VIEW @ " + string(ctx.Path()))
 
 	return ctx.Next()
 }
 
 func afterView(ctx *atreugo.RequestCtx) error {
-	logger.Info("Middleware executed AFTER VIEW")
+	logger.Info("\tMiddleware executed AFTER SPECIFIC VIEW.")
```
Comment (savsgio) [inline static_files/middlewares.go]: Add here the path as same as in `beforeView`, please

File: static_files/LICENSE [hunk 3]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (savsgio) [inline static_files/LICENSE]: Change this file with another example content and rename it, please

**Review by savsgio — CHANGES_REQUESTED** [review]
Could you change the review points?

**Comment by savsgio** [comment]
Thanks @979156 !

**Comment by yeongjukang** [comment]
@savsgio thanks for fast response!

**Pull request merged**